ddtrace/tracer.py

```diff
@@ -652,26 +683,7 @@ def current_span(self):
             return ctx.get_current_span()
         return None
 
-    def write(self, spans):
-        # type: (Optional[List[Span]]) -> None
-        """
-        Send the trace to the writer to enqueue the spans list in the agent
-        sending queue.
-        """
-        if not spans:
-            return  # nothing to do
-
-        if self.log.isEnabledFor(logging.DEBUG):
-            self.log.debug("writing %s spans (enabled:%s)", len(spans), self.enabled)
-            for span in spans:
-                self.log.debug("\n%s", span.pprint())
-
-        if not self.enabled:
-            return
-
-        spans = self.processor.process(spans)
-        if spans is not None:
-            self.writer.write(spans=spans)
+    write = _on_span_finish
```

Comment:
This probably doesn't make sense to do since the API is different (takes a span instead of a list of spans)

Reply:
Right, I'd drop this from the API.